test/parse.jl

```diff
@@ -1187,3 +1187,14 @@ module Test21607
         x
     end === 1.0
 end
+
+# Basic parsing for in place assign
+@test parse("a@b = 1") == :(a@b = 1)
```

Comment:
This kind of test isn't really effective; for example it would pass if `a@b = 1` parsed as `42`.